packages/suite/src/components/suite/ToastContainer.tsx

```diff
@@ -36,7 +36,7 @@ const StyledContainer = styled(BaseToastContainer)`
         color: ${({ theme }) => theme.TYPE_DARK_GREY};
         background: ${({ theme }) => theme.BG_WHITE};
         padding: 0px;
-        font-family: 'TT Hoves', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Helvetica Neue',
+        font-family: 'T2T1 Hoves', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Helvetica Neue',
```

Comment:
This is a good one 😆 

```suggestion
        font-family: 'TT Hoves', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Helvetica Neue',
```